aws.yaml

```diff
@@ -1040,130 +504,6 @@ spec:
 ---
 apiVersion: release.giantswarm.io/v1alpha1
 kind: Release
-metadata:
-  annotations:
-    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v8.4.1
-spec:
-  state: deprecated
-  date: 2019-09-24T11:00:00Z
-  apps: []
-  components:
-  - name: app-operator
-    version: 1.0.0
-  - name: aws-operator
-    version: 5.3.1
-  - name: cert-operator
-    version: 0.1.0
-  - name: chart-operator
-    version: 0.7.0
-  - name: cluster-operator
-    version: 0.19.0
-  - name: kubernetes
-    version: 1.14.6
-  - name: containerlinux
-    version: 2135.4.0
-  - name: coredns
-    version: 1.6.2
-  - name: calico
-    version: 3.8.2
-  - name: etcd
-    version: 3.3.13
----
-apiVersion: release.giantswarm.io/v1alpha1
-kind: Release
-metadata:
-  annotations:
-    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v8.4.0
-spec:
-  state: deprecated
-  date: 2019-08-19T16:30:00Z
-  apps: []
-  components:
-  - name: app-operator
-    version: 1.0.0
-  - name: aws-operator
-    version: 5.3.0
-  - name: cert-operator
-    version: 0.1.0
-  - name: chart-operator
-    version: 0.7.0
-  - name: cluster-operator
-    version: 0.19.0
-  - name: kubernetes
-    version: 1.14.6
-  - name: containerlinux
-    version: 2135.4.0
-  - name: coredns
-    version: 1.6.2
-  - name: calico
-    version: 3.8.2
-  - name: etcd
-    version: 3.3.13
----
-apiVersion: release.giantswarm.io/v1alpha1
-kind: Release
-metadata:
-  annotations:
-    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v8.3.0
-spec:
-  state: deprecated
-  date: 2019-08-19T10:00:00Z
-  apps: []
-  components:
-  - name: aws-operator
-    version: 5.2.1
-  - name: cert-operator
-    version: 0.1.0
-  - name: chart-operator
-    version: 0.7.0
-  - name: cluster-operator
-    version: 0.18.0
-  - name: kubernetes
-    version: 1.14.5
-  - name: containerlinux
-    version: 2135.4.0
-  - name: coredns
-    version: 1.5.1
-  - name: calico
-    version: 3.7.2
-  - name: etcd
-    version: 3.3.13
----
-apiVersion: release.giantswarm.io/v1alpha1
-kind: Release
-metadata:
-  annotations:
-    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v8.2.1
```

Comment:
Not sure about removing v8.2.1, as we still have cluster using v8.2.0.

Reply:
But the release is deprecated. Why should we bother with that?